contrib/local-environment/docker-compose-traefik.yaml

```diff
@@ -34,7 +34,7 @@ services:
   # Reverse proxy
   gateway:
     container_name: traefik
-    image: traefik:v2.11.6
+    image: traefik:v3.1.6
```

Comment:
was this tested are our traefik settings compatible with v3?

Reply:
As above.